sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -53,7 +53,15 @@ import org.apache.spark.util.Utils
 
 private[sql] object Dataset {
   def apply[T: Encoder](sparkSession: SparkSession, logicalPlan: LogicalPlan): Dataset[T] = {
```

Comment:
Thank you for your feedback. I just tried the suggested change but it was failed.